src/client/components/GameCardComponent/ButtonComponent/ButtonComponent.styles.css

```diff
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Rammetto+One&family=Roboto&display=swap');
+
+.button-component {
+  color: white;
+  font-family: 'Rammetto One', cursive;
+  margin-top: 10px;
+  background: #f26114;
+  border-radius: 5px;
+  border: none;
+  width: 335px;
```

Comment:
It will make it a bit hard to reuse the button if we set a fixed width it will not be very responsive. 

Usually, Atoms like a button should not "be aware of its surroundings" so I would make it `100%` width and let the component wrapping the button e.g. the form it will be used in control the width 👍